### PR TITLE
handler corrections

### PR DESCRIPTION
**Pull request by eesmerdag**

### CODE REVIEW

File: client/transport/webapi/request.go [hunk 1]
```diff
@@ -110,7 +110,7 @@ type ValidatorParams struct {
 	TimeFrom time.Time `json:"from"`
 	// TimeTo - the inclusive ending of the time range for registration time
 	//
-	// supposed to be sent with time to
+	// supposed to be sent with time from
```
Comment (lukanus) [inline client/transport/webapi/request.go:113]: Good spot! But this should be for line 107 :)

Comment (eesmerdag) [reply client/transport/webapi/request.go:113]: maybe it is my english but what i wanted to say here: TimeTo should be sent with TimeFrom. And this message is for TimeTo so it should be correct change 🤔

Comment (lukanus) [reply client/transport/webapi/request.go:113]: oh with "with time from" got it. sorry

File: store/postgresql/validator_statistics.go [hunk 2]
```diff
@@ -87,8 +93,8 @@ func (d *Driver) GetValidatorStatisticsTimeline(ctx context.Context, params stru
 		`SELECT id, created_at, validator_id, amount, block_height, time, statistic_type
 			FROM validator_statistics
 			WHERE
-				validator_id = $1 AND statistic_type = $2
-			ORDER BY block_height DESC`, params.ValidatorID, params.Type)
+				validator_id = $1 AND statistic_type = $2 AND time BETWEEN $3 AND $4 
+			ORDER BY block_height DESC`, params.ValidatorID, params.Type, params.TimeFrom, params.TimeTo)
```
Comment (lukanus) [inline store/postgresql/validator_statistics.go:97]: Good you've added that but shouldn't that be optional?

**Review by lukanus — APPROVED** [review]
Good job!

**Pull request merged**